src/strategies/multidelegation/utils.ts

```diff
@@ -0,0 +1,211 @@
+import { getAddress } from '@ethersproject/address';
+import {
+  getDelegatesBySpace,
+  getSnapshots,
+  subgraphRequest
+} from '../../utils';
+
+export async function getPolygonDelegatesBySpace(
+  space: string,
+  snapshot = 'latest'
+) {
+  const spaceIn = ['', space];
+  if (space.includes('.eth')) spaceIn.push(space.replace('.eth', ''));
+
+  const PAGE_SIZE = 1000;
+  let result: { delegator: string; space: string; delegate: string }[] = [];
+  let page = 0;
+  const params: any = {
+    delegations: {
+      __args: {
+        where: {
+          space_in: spaceIn
+        },
+        first: PAGE_SIZE,
+        skip: 0
+      },
+      delegator: true,
+      space: true,
+      delegate: true
+    }
+  };
+  if (snapshot !== 'latest') {
+    params.delegations.__args.block = { number: Number(snapshot) };
+  }
+
+  while (true) {
+    params.delegations.__args.skip = page * PAGE_SIZE;
+
+    const pageResult = await subgraphRequest(
+      'https://api.thegraph.com/subgraphs/name/1emu/multi-delegation-polygon',
+      params
+    );
+    const pageDelegations = pageResult.delegations || [];
+    result = result.concat(pageDelegations);
+    page++;
+    if (pageDelegations.length < PAGE_SIZE) break;
+  }
+
+  return result;
+}
+
+export async function getMultiDelegations(
+  space: string,
+  network: string,
+  snapshot?: string
+): Promise<Map<string, string[]>> {
+  const delegatesBySpace = await getPolygonDelegatesBySpace(space, snapshot);
+
+  return delegatesBySpace.reduce((accum, delegation) => {
+    const delegator = getAddress(delegation.delegator);
+    const delegate = getAddress(delegation.delegate);
+    const existingDelegates = accum.get(delegator) || [];
+    accum.set(delegator, [...existingDelegates, delegate]);
+    return accum;
+  }, new Map<string, string[]>());
+}
+
+export async function getLegacyDelegations(
```

Comment:
Maybe `getSingleDelegations`, `getOneToOneDelegations` or `getSnapshotDelegations` or something like that?